### DISCUSSION TITLE
Is it possible to migrate/bootstrap from an existing redis?

### DISCUSSION THREAD

**Discussion by jonathon2nd** [OT-CONTAINER-KIT/redis-operator#865] [Q&A] [answered]
We have been using spotahome/redis-operator for our preprod environments and were prepping for production migration in the future. Though the status of the project is now [up in the air](https://github.com/spotahome/redis-operator/issues/677).

Is it possible to create a redis replication deployment pointed to an existing redis master?

https://github.com/spotahome/redis-operator?tab=readme-ov-file#bootstrapping-from-pre-existing-redis-instances

**Comment by drivebyer** [comment, marked as answer]
Currently, this operator does not support this feature. However, this feature would be a great addition for this operator!
